frontend/src/components/Pages/MissionPage/TaskOverview/TaskTable.tsx

```diff
@@ -35,23 +22,18 @@ export const TaskTable = ({ tasks }: TaskTableProps) => {
     const { TranslateText } = useLanguageContext()
 
     return (
-        <>
-            <StyledTable>
-                <StyledTableCaptionGray>
-                    <StyledTypography variant="h2">{TranslateText('Tasks')}</StyledTypography>
-                </StyledTableCaptionGray>
-                <Table.Head>
-                    <Table.Row>
-                        <StyledTableCell>#</StyledTableCell>
-                        <StyledTableCell>{TranslateText('Tag-ID')}</StyledTableCell>
-                        <StyledTableCell>{TranslateText('Description')}</StyledTableCell>
-                        <StyledTableCell>{TranslateText('Inspection Types')}</StyledTableCell>
-                        <StyledTableCell>{TranslateText('Status')}</StyledTableCell>
-                    </Table.Row>
-                </Table.Head>
-                <StyledTableBody>{tasks && <TaskTableRows tasks={tasks} />}</StyledTableBody>
-            </StyledTable>
-        </>
+        <StyledTable>
```

Comment:
Not necessary to solve in this PR but consider utilizing the available room on the right for the task table

<img width="446" alt="image" src="https://github.com/user-attachments/assets/d382e2fb-eaae-4766-a8c1-7b5e4f51cb08" />